package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "start:example": "bash -c 'RUNNER=\"./examples/$0/run.js\"; KA_LOG_LEVEL=silly NODE_ENV=development babel-watch \"$RUNNER\"'",
```

Comment:
Query: I toyed with this locally, but what do you think about changing `start:example` to just `example`? Since I've been using this a lot, it's been quicker to type :grinning:

Reply:
Sure thing!